src/cpp/rtps/builtin/discovery/database/DiscoveryDataBase.hpp

```diff
@@ -331,9 +331,9 @@ class DiscoveryDataBase
         return virtual_topic_;
     }
 
-    int get_etinties_updated_and_reset()
+    int new_entities()
     {
-        return entities_updated_.exchange(0);
+        return new_entities_.exchange(0);
```

Comment:
If this function reset this value, I think it should be written in its name. Otherwise I would do a get function and a reset function separately.

Reply:
In my mind, if it's called `new_updates`, it means since the last time you checked, else they are not new to you. It could also be called `updates_since_last_checked`. I'm open to suggestions!

Reply:
That makes sense, but also it could be misunderstood because it is also the name of the variable, and so it is the getter/setter. I'd rather prefer `updates_since_last_checked`, just in case.